dss/operations/storage.py

```diff
@@ -108,6 +108,26 @@ def process_key(self, key):
                                       file_metadata_content_type=file_metadata['content-type'],
                                       blob_content_type=blob_content_type))
 
+@storage.action("repair-file-blob-metadata",
+                arguments={"--entity-type": dict(default="file", choices=["file"])})
+class repair_blob_content_type(StorageOperationHandler):
+    """
+    Make blob metadata consistent with file metadata.
+    """
+    def process_key(self, key):
```

Comment:
Should there be a function to crawl through the entire bucket and make repairs?

Reply:
The operator should decide what to do based on the output of `scripts/dss-ops.py storage verify-file-blob-metadata`.

Once a list of repair candidates is generated, scripting may be used to either call the repair commands locally or SQS forward them for Lambda execution:
```
for key in $keys; do
  scripts/dss-ops.py storage repair-file-blob-metadata --replica $replica --keys $key --forward-to-lambda
done
```

Reply:
If you use the command above on only one replica as it seems to be doing, will the changes sync?